lib/gringotts/gateways/cams.ex

```diff
@@ -58,10 +71,7 @@ defmodule Gringotts.Gateways.Cams do
           |> add_invoice(money, options)
```

Comment:
@gopalshimpi indentation is I guess 2 tabs with 4 spaces each. use 2 spaces and covert all tabs to spaces. again review your changes in the browser.

Reply:
@pkrawat1 I have made necessary changes which pointed by you

Reply:
Remove address struct if it's unused.